src/main/java/org/vietabroader/view/verifier/SheetVerifier.java

```diff
@@ -0,0 +1,25 @@
+package org.vietabroader.view.verifier;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class SheetVerifier extends InputVerifier {
+    @Override
+    public boolean verify(JComponent input) {
```

Comment:
The goal of this class seems to only be unifying the `shouldYieldFocus` method. Remove this method override if it does not serve any purpose, and make this class abstract.